core/src/main/scala/org/apache/spark/rdd/RDD.scala

```diff
@@ -1507,7 +1507,8 @@ abstract class RDD[T: ClassTag](
     val r = this.mapPartitions { iter =>
       val text = new Text()
       iter.map { x =>
-        text.set(x.toString)
+        val value = if (x != null) x.toString else "Null"
+        text.set(value)
```

Comment:
I would simply just add an assert or require.

Reply:
done! Thank you @HyukjinKwon